code/modules/nifsoft/nif.dm

```diff
@@ -69,6 +70,7 @@ You can also set the stat of a NIF to NIF_TEMPFAIL without any issues to disable
 
 	//Put loaded data here if we loaded any
 	save_data = islist(load_data) ? load_data.Copy() : list()
+	examine_msg = save_data["examine_msg"] || "There's a certain spark to their eyes."
```

Comment:
Does this mean that clearing the message will get reset to spark after next load?

Reply:
Yes, fixed.